packages/test/src/config.js

```diff
@@ -13,7 +13,7 @@ module.exports = {
   rootDir: cwd,
   roots: [`${cwd}/webpack/`],
   verbose: true,
-  automock: true,
+  automock: false,
```

Comment:
**I am canceling the auto mocking because we did it entirely wrong.**

We used `unmockedModulePathPatterns: ['react', 'node_modules/']` to tell jest what **not to mock**.
The result of that was:
1. Do not mock any node_modules, while `automock` are mostly to `automock` node_modules :/
2. We had `react` there, it makes sense not to mock react but it's already a node module so we should be cover already because of the `'node_modules/'`.
The side effect of that was, not to mock anything in the `react_app` folder :/

In other words, the result of this config was: **"Mock all the `webpack/assets/javascript` but don't mock `webpack/assets/javascript/react_app`"**.

But, because we don't want to mock the files in `webpack/assets/javascript`, in every test in this folder we would use `jest.unmock` to be able to test those files.
For example: 
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/foreman_config_reports_modal_diff.test.js#L5
https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/foreman_toast_notifications.test.js#L7

After understanding this mess, I think we should just cancel the `automock` and revisit that in the future.

Reply:
makes sense, will this change cause any of the existing tests to break (e.g. because they rely on previously auto-mocked functions)?